spinnaker.md

```diff
@@ -0,0 +1,268 @@
+# Deploying services onto EKS with Spinnaker
+
+## Install Halyard
+
+Halyard is the command line interface for managing and deploying Spinnaker. Although a Helm chart is available for Spinnaker, Halyard is still considered the best way to install Spinnaker.
+
+### Install on OSX
+
+**Prerequisites**
+
+Be sure you have Java 1.8 installed before continuing.  To determine whether you have Java 1.8 installed, from the command line, type:
+
+```
+java -version
+```
+
+If you don't have it installed, Java 1.8 can be download for OSX from [http://www.oracle.com/technetwork/java/javase/downloads/jdk8-downloads-2133151.html](http://download.oracle.com/otn-pub/java/jdk/8u171-b11/512cd62ec5174c3487ac17c61aaa89e8/jdk-8u171-macosx-x64.dmg).
+
+Once installed, run java_home to verify the Java 1.8 home:
+
+```
+/usr/libexec/java_home -v 1.8
+```
+
+**Download the Installer**
+
+```
+curl -O https://raw.githubusercontent.com/spinnaker/halyard/master/install/macos/InstallHalyard.sh
+```
+
+Install Halyard: 
+
+```
+sudo bash InstallHalyard.sh
+```
+
+If you get an error about not finding Java 1.8, manually set the environment:
+
+```
+export JAVA_HOME=/usr/libexec/java_home -v 1.8
+export PATH=$PATH:$JAVA_HOME/bin
+```
+
+and pass the same environment to launch the installer:
+
+```
+sudo JAVA_HOME=`/usr/libexec/java_home -v 1.8` PATH=$PATH:$JAVA_HOME/bin bash InstallHalyard.sh
+```
+
+**Verify Installation**
+
+```
+hal -v
+```
+
+Move `hal` into your `$PATH`
+
+## Create a Github repository
+
+Later in this exercise you will be creating a Spinnaker pipeline that runs when code is pushed to a Github repository called spin-kub-repo.
+ 
+Login to Github and fork the [lwander/spin-kub-demo](https://github.com/lwander/spin-kub-demo) repository to your Github account.  
```

Comment:
Purpose of this repo is not clear

Reply:
It says, "Later in this exercise you will be creating a pipeline that runs when code is pushed to a repository called spin-kub-repo."  You don't think that's clear enough?